lib/usd/translators/shading/usdLambertWriter.cpp

```diff
@@ -111,11 +111,16 @@ void PxrUsdTranslators_LambertWriter::WriteSpecular(const UsdTimeCode& usdTime)
         .CreateInput(PxrMayaUsdPreviewSurfaceTokens->RoughnessAttrName, SdfValueTypeNames->Float)
         .Set(1.0f, usdTime);
 
-    // Using specular workflow, but with default black specular color.
+    // Using specular workflow, but enforced black specular color.
     shaderSchema
         .CreateInput(
             PxrMayaUsdPreviewSurfaceTokens->UseSpecularWorkflowAttrName, SdfValueTypeNames->Int)
         .Set(1, usdTime);
+
+    shaderSchema
+        .CreateInput(PxrMayaUsdPreviewSurfaceTokens->SpecularColorAttrName, SdfValueTypeNames->Color3f)
+        .Set(GfVec3f(0.0f, 0.0f, 0.0f), usdTime);
```

Comment:
We need the value to be "set" instead of "default" to zero if we want to read it back correctly on import.